test/app-luatest/gh_4975_errors_in_init_scripts_are_stripped_test.lua

```diff
@@ -0,0 +1,22 @@
+local t = require('luatest')
+local fio = require('fio')
+local justrun = require('test.justrun')
+
+local g = t.group('gh-4975')
+
+--[[
+Require a non-existent module with a long enough name to get an error message,
+which is longer than 1000 characters, like:
+
+LuajitError: module 'aaa<...>aaa' not found:
+    no file '/usr/local/lib/lua/5.1/aaa<...>aaa.so'
+    no file '/usr/local/lib/lua/5.1/override/aaa<...>aaa.so'
+    <...>
+]]
+g.test_long_init_error = function(cg)
+    local dir = fio.tempdir()
```

Comment:
This temporary directory is never removed (and it is why I usually use `test.treegen` for such cases).

However, we can run the process in the current directory (pass `'.'` instead of `dir` to `justrun.tarantool()`).

Reply:
```diff
--- a/test/app-luatest/gh_4975_errors_in_init_scripts_are_stripped_test.lua
+++ b/test/app-luatest/gh_4975_errors_in_init_scripts_are_stripped_test.lua
@@ -14,9 +14,8 @@ LuajitError: module 'aaa<...>aaa' not found:
     <...>
 ]]
 g.test_long_init_error = function(cg)
-    local dir = fio.tempdir()
     local module_name = string.rep('a', 500)
     local args = {'-l', module_name, '-e', '""'}
-    local res = justrun.tarantool(dir, {}, args, {stderr = true, nojson = true})
+    local res = justrun.tarantool('.', {}, args, {stderr = true, nojson = true})
     t.assert_gt(string.len(res.stderr), 1000, 'error is longer than 1000 chars')
 end
```